### PR TITLE
Fix GitHub issue #36390: update Source-to-image information

### PR DESCRIPTION
**Pull request by lbscorpio**
@vikram-redhat This PR is to fix the issue#36390, minimum version that the change applies to should be 4.5 in my opinion, could you arrange someone to review them? Thanks.

Major changes on my commit:
1. Add additional S2I images
2. Add detailed procedure to find the available S2I images

### CODE REVIEW

File: openshift_images/using_images/using-s21-images.adoc [hunk 1]
```diff
@@ -9,14 +9,22 @@ You can use the link:https://access.redhat.com/documentation/en-us/red_hat_softw
...
 * Node.js
 * Perl
 * PHP
 * Python
 * Ruby
 
-S2I images are available for you to use directly from the {product-title} web UI by selecting *Catalog* -> *Developer Catalog*.
+S2I images are available for you to use directly from the {product-title} web console by following procedure:
+
+. Log in to the OpenShift Container Platform web console using your login credentials. The default view for the OpenShift Container Platform web console is the *Administrator* perspective.
```
Comment (bmcelvee) [inline openshift_images/using_images/using-s21-images.adoc:23]: Requested edit, for every instance of OpenShift Contain Platform:
s/OpenShift Contain Platform/{product-title} 

If you prefer, I can open a follow-up PR to make this change.

**Comment by bmcelvee** [comment]
@lbscorpio thank you for opening this PR! I have one requested edit. 

@wewang58 would you mind giving this PR a review, please? Thank you!

**Comment by lbscorpio** [comment]
@bmcelvee Thanks for your feedback, I have updated it on my commit.

File: openshift_images/using_images/using-s21-images.adoc (hunk 1 above)
Comment (lbscorpio) [reply openshift_images/using_images/using-s21-images.adoc]: Updated, thanks for the feedback.

**Review by bmcelvee — APPROVED** [review]

**Comment by bmcelvee** [comment]
LGTM! Thank you!

**Pull request merged**